engine/src/main/java/org/terasology/rendering/gltf/GLTFAnimationFormat.java

```diff
@@ -159,7 +160,7 @@ private MeshAnimationData loadAnimation(GLTF gltf, GLTFAnimation animation, List
             frames.add(frame);
         }
 
-        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, AABB.createEmpty());
+        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, new AABBf());
```

Comment:
`new AABBf()` creates with (minX, minY, minZ)=(+inf, +inf, +inf) and (maxX, maxY, maxZ)=(-inf, -inf, -inf)